tests/parametric/test_parametric_workflow.py

```diff
@@ -130,28 +138,36 @@ def test_parametric_workflow(monkeypatch: pytest.MonkeyPatch):
     assert "New Study" in solver_session.parametric_studies
     del solver_session.parametric_studies[study1_name]
     assert len(solver_session.parametric_studies) == 1
-    project_filename = Path(save_path) / "static_mixer_study.flprj"
-    solver_session.file.parametric_project.save_as(
-        project_filename=str(project_filename)
-    )
-    assert project_filename.exists()
+    project_filename = "static_mixer_study.flprj"
+    solver_session.file.parametric_project.save_as(project_filename=project_filename)
```

Comment:
Does it save at the mount path dir if we pass something like `/tmpdir/static_mixer_study.flprj` here?

Reply:
It should work, yes. 

I tried to minimize the amount of times that we have to check whether a container run was specified, do you think it would be better to, rather than use relative paths with `.flprj.cffdb/...`, check if running in a container and then using absolute paths with the known container mount path?

Reply:
When we are using `.flprj.cffdb/...`, we are basing the test on a defective Fluent behaiour. So passing an absolute path is better in this case IMHO. The container mount path can be stored in a session property, then we can remove the hardcoding of `/tmpdir` too.

Reply:
That makes sense, and I agree. I did not realize the Fluent `.flprj.cffdb/...` pathing was considered a mistake, but I can see where you are coming from, that is definitely very unintuitive behavior (same as in the other instances where Fluent changes its own working directory without the user necessarily knowing this).

Going to suggest some changes in light of this. 

By the way, even if we store the container mount path in a session connection property, we'd still need to specify this path every time we are sending commands to Fluent from PyFluent, as Fluent itself is not aware of the container mount path difference, only PyFluent is (unless we rework how Fluent works). Also, we can currently use the `DEFAULT_CONTAINER_MOUNT_PATH` const as it is a little less hardcoded than specifying `/tmpdir` directly.